tests/testAutopas/tests/selectors/tuningStrategy/FullSearchTest.cpp

```diff
@@ -19,31 +19,31 @@ TEST_F(FullSearchTest, testSearchSpaceEmpty) {
 TEST_F(FullSearchTest, testTune) {
```

Comment:
shouldn't this be refactored similarly to the predictive tuning test?

Reply:
feel free ;)

Reply:
done myself